integration-tests/hdwallet/ssh/id_rsa

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
what's the purpose of this? is it freshly generated for test-only?

Reply:
yes 
it's testing only